<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentRecordsDialog</class>
 <widget class="QDialog" name="StudentRecordsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open File</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui.qrc">
    <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>233</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelId">
      <property name="text">
       <string>ID</string>
      </property>
      <property name="buddy">
       <cstring>lineEditId</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditId"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Name</string>
      </property>
      <property name="buddy">
       <cstring>lineEditName</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelAddress">
      <property name="text">
       <string>Address</string>
      </property>
      <property name="buddy">
       <cstring>plainTextAddress</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="plainTextAddress"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelAverage">
      <property name="text">
       <string>Average</string>
      </property>
      <property name="buddy">
       <cstring>lineEditAverage</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditAverage"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>241</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonSeek">
      <property name="text">
       <string>Seek</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonWrite">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonPrev">
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonAddRecord">
      <property name="text">
       <string>Add Record</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonDelete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="buttonNext">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StudentRecordsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StudentRecordsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
